README.md

```diff
@@ -0,0 +1,117 @@
+# Web Test Transformer
+
+- [Web Test Transformer](#web-test-transformer)
+  - [Overview](#overview)
+  - [Setup](#setup)
+  - [Usage](#usage)
+    - [Dry Run](#dry-run)
+    - [CLI Flags](#cli-flags)
+      - [Required Flags](#required-flags)
+  - [Verification](#verification)
+    - [All PASS Requirement](#all-pass-requirement)
+  - [Issues](#issues)
+
+## Overview
+Web Test Transformer is designed to transform HTML tests in Chromium source from the legacy
+[js-test.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/js-test.js)
+framework to the current
+[testharness.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/testharness.js)
+framework.
+
+
+View the design document at [go/web-test-transformer](go/web-test-transformer) (internal Google access only).
+
+## Setup
+
+`$git clone git@github.com:Ecosystem-Infra/web-test-transformer.git`
+
+`$npm install` to install dependencies.
+
+## Usage
+
+Requirements:
+- You must be in `chromium/src/third_party/blink/web_tests` (for verification)
+- You must have chromium built. (for verification)
+- You are not running the tool on Windows (no path separator support). 
+
+
+This tool does its best job to meaningfully log all of the major actions it takes, as well as any 
+problems or errors it encounters along the way. For a large directory there will be a lot of output,
+but it should be easily searchable. 
+
+For a file, the tool will attempt to transform that file
```

Comment:
I think we could refactor some of this section into a `Functional Overview` and an `Example output` section. And maybe rename your existing "Overview" to "Introduction" :)

Move to overview:
- tool works on dir or file
- which kinds of files are handled
- tool is best effort, gives a searchable summary of what it could/couldn't do

Then also create an output section with some examples to make it clearer what kind of things to search for

Also, consider whether some details are relevant to the user - ie: the point about not doing batch-transform and batch-verify? Is that meant to identify a specific limitation? Maybe it would be more clear in an example section.